### PR TITLE
Changed PIPECE to work with FODESystem with parameters. Also made PIP…

### PR DESCRIPTION
**Pull request by jClugstor**
…ECE return an FODESolution

Fix: #40

### CODE REVIEW

File: src/fodesystem/PIPECE.jl [hunk 1]
```diff
@@ -151,12 +151,12 @@ function solve(prob::FODESystem, h, ::PECE)
     end
 
     t = t[1:N+1]; y = y[:, 1:N+1]
-    return t, y
+    return FODESolution(t,y)
```
Comment (ErikQQY) [inline src/fodesystem/PIPECE.jl:154]: The `FODESolution` is now only support single term FODE and Multi terms FODE, the solution types for `FODESystem` has not been implemented so far.
```suggestion
    return t, y
```

**Review by ErikQQY — CHANGES_REQUESTED** [review]
Thank you for your contributions❤️!! And sorry for the errors. The solution types for FODESystem has not been implemented, so I think we can focus on fixing the parameter in the construction of FODESystem.

**Comment by jClugstor** [comment]
Oh I see, what would need to be done to implement the solution type for FODESystem?

**Comment by ErikQQY** [comment]
I think the solution type for ```FODESystem``` is important since there are many solvers available for system of fractional differential equations, but right now the solution is stored and returned as a matrix, which is not users friendly, the future works would be adding the plot recipe for this type, like what [OrdinaryDiffEq.jl](https://github.com/SciML/OrdinaryDiffEq.jl) does:

```julia
plot(sol)
plot(sol, vars=(1, 2, 3))
plot(sol, vars=(1, 2))
```

**Pull request merged**